.github/profile/README.md

```diff
@@ -0,0 +1,120 @@
+# Welcome to Eclipse Open Regulatory Compliance Working  <img src="https://avatars.githubusercontent.com/u/189788385?s=200&v=4" width="200px"/>
+
+
+## About The Working Group
+
+New government regulations on the software industry will impact open source communities. The community should support their successful implementation. This working group is intended to bring together key stakeholders from industry, small and medium enterprise (SME), research, and open source foundations to work with government in forging specifications that will enable industry to continue to leverage open source through the software supply chain to meet those regulatory requirements, and in turn will enable the open source projects to better meet industry's needs in this regard.
+
+
+## Our Goal
+
+ Working within the Eclipse Foundation framework for developing Specifications, the working group will:
+
+* Define software development processes - either current or adapted - which meet the regulatory compliance standards set forth by regulatory authorities. This is achieved through:
+    * Identifying best practices, process and tools which may already exist within the open source ecosystem
+    * Identifying gaps and resources that may fill those gaps
+* Identify and assess relevant regulatory requirements applicable to software development, including but not limited to cybersecurity, AI, and industry-specific directives.
+* Establish Special Interest Groups (SIGs) for domain-specific work (i.e. Cyber and AI).
+* Compile guidelines and best practices for integrating compliance considerations into all stages of the software development lifecycle, from planning and design to deployment and maintenance.
+* Collaborate with legal experts, industry stakeholders, regulatory bodies, and government officials to ensure alignment with current and evolving regulatory frameworks.
+* Provide a forum where feedback can be garnered from government with respect to the specifications being developed, and that all interested stakeholders can meet when and how appropriate to better ensure the specifications meet the objectives established by government.
+* Foster transparency and accountability within the open source community by promoting adherence to regulatory standards.
+* Identify gaps in educational resources and training materials to support open source developers in understanding and implementing compliant development practices.
+* With staff support, monitor regulatory updates and emerging trends to continuously refine and update the established processes and guidelines.
+* Promote the adoption of the resulting specifications to various relevant standardisation bodies.
+* Through its member organisations, promote the adoption of compliant software development practices across industry through advocacy, outreach, and knowledge-sharing initiatives.
+
+
+## How to Get Involved
```

Comment:
I'd add our key resources there:

The SIG, the CRA hub and maybe the FAQ?

Reply:
see: https://github.com/orcwg/.github/pull/1/commits/fd5876f97187bb12ba9f1bcd3a2b55c30fb111f4